packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Navigation.tsx

```diff
@@ -48,22 +49,38 @@ const Navigation = (props: NavigationProps): JSX.Element => {
     );
   }
 
+  const [isOpenMenu, openMenu] = useState(false);
+  const handleToggleMenu = (flag?: boolean): void => {
+    if (flag !== undefined) {
+      return openMenu(flag);
+    }
+
+    openMenu(!isOpenMenu);
+  };
   const list = Object.values(props.fields)
     .filter((element) => element)
     .map((element: Fields, key: number) => (
-      <NavigationList key={`${key}${element.Id}`} fields={element} />
+      <NavigationList key={`${key}${element.Id}`} fields={element} handleClick={() => handleToggleMenu(false)} />
```

Comment:
You provide handleClick as a function which doesn't have parameters, in this case on the line 108 it doesn't make sense to provide anything, it will not accept anything